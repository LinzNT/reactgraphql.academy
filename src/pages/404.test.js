import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

import NotFoundPage from './404'

describe('<NotFoundPage />', () => {
  it('should render component', () => {
    const wrapper = shallow(<NotFoundPage />)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
