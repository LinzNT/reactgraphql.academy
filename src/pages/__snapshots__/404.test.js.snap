// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundPage /> should render component 1`] = `
<div>
  <h1>
    NOT FOUND
  </h1>
  <p>
    You just hit a route that doesn't exist... the sadness.
  </p>
</div>
`;
